onion_store.go

```diff
@@ -44,21 +44,20 @@ func onionPath(onionType tor.OnionType) (string, error) {
 // onionStore is an etcd-based implementation of tor.OnionStore.
 type onionStore struct {
 	*clientv3.Client
+	OnionType tor.OnionType
```

Comment:
nit: un-export since the struct itself is not exported either? Also I'd add a newline between embedded and non-embedded struct members.

Reply:
The alternative would be to remove V2 support completely as that's not supported by the Tor network anymore anyway (since October last year I think).

Reply:
I've opted to remove V2 support completely.